### PR TITLE
Update Design on Market Sidebar Block

### PR DESCRIPTION
**Pull request by jm90m**
Fixes: #1102.

Changes:
- Move CryptoCharts API requests to state/reducers
- Use Recharts instead of react-trend
- Update Styles on title for all sidebar block components
- Match New Design of crypto chart

screen shot 2017-12-19 at 11 36 50 am

### CODE REVIEW

File: src/client/activity/UserActivitySearch.js [hunk 1]
```diff
@@ -221,61 +221,62 @@ class UserActivitySearch extends React.Component {
...
-                  ? <i className="iconfont icon-offline" />
-                  : <i className="iconfont icon-addition" />}
-              </span>
+        <div className="SidebarContentBlock__content">
+          <div className="UserActivitySearch__filters">
+            <div className="UserActivitySearch__filters__container">
+              <div
+                role="presentation"
+                className="UserActivitySearch__filters__title"
+                onClick={() => this.toggleFilterSection('showGeneral')}
```
Comment (Sekhmet) [inline src/client/activity/UserActivitySearch.js]: Can we have handler for this?

File: src/client/activity/UserActivitySearch.js [hunk 2]
```diff
@@ -221,61 +221,62 @@ class UserActivitySearch extends React.Component {
...
-              <span className="UserActivitySearch__filters__title__icon">
-                {showFinance
-                  ? <i className="iconfont icon-offline" />
-                  : <i className="iconfont icon-addition" />}
-              </span>
+            <div className="UserActivitySearch__filters__container">
+              <div
+                role="presentation"
+                className="UserActivitySearch__filters__title"
+                onClick={() => this.toggleFilterSection('showFinance')}
```
Comment (Sekhmet) [inline src/client/activity/UserActivitySearch.js]: And this?

File: src/client/activity/UserActivitySearch.js [hunk 3]
```diff
@@ -221,61 +221,62 @@ class UserActivitySearch extends React.Component {
...
-              <span className="UserActivitySearch__filters__title__icon">
-                {showRewards
-                  ? <i className="iconfont icon-offline" />
-                  : <i className="iconfont icon-addition" />}
-              </span>
+            <div className="UserActivitySearch__filters__container">
+              <div
+                role="presentation"
+                className="UserActivitySearch__filters__title"
+                onClick={() => this.toggleFilterSection('showRewards')}
```
Comment (Sekhmet) [inline src/client/activity/UserActivitySearch.js]: And this?

File: src/client/activity/UserActivitySearch.js (hunk 1 above)
Comment (jm90m) [reply src/client/activity/UserActivitySearch.js]: yeah looks these were old code

**Comment by jm90m** [comment]
@Sekhmet just made handlers for each of them in `UserActivitySearch`

**Pull request merged**